lib/Myriad/Commands.pm

```diff
@@ -63,36 +65,29 @@ async method service (@args) {
 
 async method rpc ($rpc, @args) {
     try {
-        my $response = await $myriad->rpc_client->call_rpc($myriad->config->service_name->as_string, $rpc, @args);
+        my $service = Myriad::Service::Remote->new(myriad => $myriad, service_name => $myriad->registry->make_service_name($myriad->config->service_name->as_string));
```

Comment:
there would be more than one service - instead, we should check the existing services, throw an exception if there's more than one available, then use the first (and only) name in the list

Reply:
hmm, didn't understand this well enough, but in any case, I think it's something related to config handling so it was wrong even before my change.